srv/server.py

```diff
@@ -69,14 +71,13 @@ async def get_shapes(beg_time: int = None, end_time: int = None) -> Shapes:
                     "y": rect[3],
                     "w": rect[4],
                     "h": rect[5],
+                    "created_at": rect[6],
+                    "type": "rect",
                 }
             )
 
-        resp = Shapes(
-            circles=circle_resp,
-            rectangles=rect_resp,
-        )
-
+        resp = circle_resp + rect_resp
```

Comment:
We'll need to change the `Shapes` class as well to match the new format (see `response_model` at the top of the endpoint).

Reply:
Replacing `Shapes` class with `list[Shape]` - a list of a new object that both `Circle` and `Rectangle` extend.